### ISSUE TITLE
Reported baseline result

### ISSUE THREAD

**Issue by aminpdik** [chunbolang/BAM#2]
Thank you for the interesting paper @chunbolang . Specifically showing a **simple** meta learner with the help of a supervised network can produce state-of-the-art results.

My first question is the reported result for baseline in the table1.

Is the baseline just a meta learner network (without base learner and ensemble module) on top of a frozen backbone (Pretrained by weights of a segmentation network trained on base classes)? and it is only trained by loss_meta on base classes?

![image](https://user-images.githubusercontent.com/88690656/159136604-851d62e6-3375-4054-94ce-b2f3c7983678.png)

As you know the same meta leaner in the previous works on top of frozen backbone **pretrained by the Imagenet**  can produce for example ~%55 MIOU on 1 shot setting of Pascal. In your case, it is reported 65.4 for **baseline**. I want to know this big boost was achieved because of using frozen backbone Pretrained by weights of a segmentation network trained on base classes?

**Comment by chunbolang** [comment]
@aminpdik 
Hi, thanks for your attention!

1. We do not train the meta learner _separately_ to obtain baseline results. In the meta-training stage, the meta learner and the ensemble module are jointly optimized. We take the former's output as the baseline result and the latter's output as the final result. A frozen backbone trained pre-trained by the base learner is employed, as you mentioned; however, both loss functions (_i.e._, L_meta & L_final) will affect weight updates.

2. That's a good question. Aside from identifying irrelevant regions through base learners, another focus of our work is to explore the role of base datasets, which could provide us with far more than a series of meta-tasks (episodes). In this work, we additionally train a traditional segmentation model with a stronger backbone on base classes, and perhaps more interesting ways will be explored by the community in future research. The improvement of the meta learner is related to both the backbone network and joint optimization (see point 1), and the former might be **dominant**. But we currently are not clear about the potential of simply using our backbone network for separate training, and please inform me if there is any progress. Furthermore, we build the meta learner based on [PFENet](https://github.com/dvlab-research/PFENet), which could acquire ~61% mIoU on PASCAL 1-shot. 

Regards,

**Comment by aminpdik** [comment]
Thanks for answering my question. I am exploring your paper and code. I will be in touch with you if you don't mind.

**Comment by chunbolang** [comment]
Please feel free to contact us~!

**Comment by 66Kevin** [comment]
> Thank you for the interesting paper @chunbolang . Specifically showing a **simple** meta learner with the help of a supervised network can produce state-of-the-art results.
> 
> My first question is the reported result for baseline in the table1.
> 
> Is the baseline just a meta learner network (without base learner and ensemble module) on top of a frozen backbone (Pretrained by weights of a segmentation network trained on base classes)? and it is only trained by loss_meta on base classes?
> 
> ![image](https://user-images.githubusercontent.com/88690656/159136604-851d62e6-3375-4054-94ce-b2f3c7983678.png)
> 
> As you know the same meta leaner in the previous works on top of frozen backbone **pretrained by the Imagenet** can produce for example ~%55 MIOU on 1 shot setting of Pascal. In your case, it is reported 65.4 for **baseline**. I want to know this big boost was achieved because of using frozen backbone Pretrained by weights of a segmentation network trained on base classes?

Hi,
Thank you for your question. Can I know which previous work uses the same meta learner whose backbone pretrained by the Imagenet?

Regards
